composer.json

```diff
@@ -40,10 +40,10 @@
             "type": "package",
             "package": {
                 "name": "danskernesdigitalebibliotek/dpl-react",
-                "version": "2024.50.0",
+                "version": "0.0.0-dev",
```

Comment:
These changes in composer.json & composer.lock shouldn't be coming in.

Reply:
It is a temporary commit for for demoing purposes, as you can see in the commit message